src/main/java/CinemaRoomRestService/RestService/api/mapper/SeatMapper.java

```diff
@@ -0,0 +1,46 @@
+package CinemaRoomRestService.RestService.api.mapper;
+
+import CinemaRoomRestService.RestService.api.dto.StatisticsDTO;
+import CinemaRoomRestService.RestService.domain.entity.CinemaRoom;
+import CinemaRoomRestService.RestService.domain.entity.Seat;
+import CinemaRoomRestService.RestService.domain.entity.Statistics;
+import CinemaRoomRestService.RestService.domain.entity.TokenOfSeat;
+import CinemaRoomRestService.RestService.api.dto.CinemaRoomDTO;
+import CinemaRoomRestService.RestService.api.dto.SeatDTO;
+import CinemaRoomRestService.RestService.api.dto.TokenOfSeatDTO;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SeatMapper {
+
+    public CinemaRoomDTO toCinemaRoomDTO(CinemaRoom cinemaRoom) {
+        return CinemaRoomDTO
+                .builder()
```

Comment:
Code style, call builder on the same line please

Reply:
Fixed